meta/kwoptarg.rb

```diff
@@ -1,11 +1,11 @@
 Mutest::Meta::Example.add :kwarg do
-  source 'def foo(bar: baz); end'
+  source 'def foo(bar: {}); end'
```

Comment:
Ah. I usually add a new example since otherwise you run the risk of editing a test that kills other mutations